app/script/controllers/my.profile.controller.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Created by sa on 2016.12.22..
+ */
+(function () {
+    angular.module('app')
+        .controller('myProfileController',function (apiService, $scope, $location) {
+                apiService.getUserId()
+                    .then(function (response) {
+                        $scope.myProfile=response.data;
+                        $scope.right=response.data.rights;
+                    });
+                $scope.myProfileUpdate=function () {
+
+                    apiService.myProfileUpdate($scope.myProfile)
+                        .then(function (response) {
+                            $scope.myProfile=response.data;
+                        })
+                };
+
+            $scope.checkPhone = function(data) {
+                var re = new RegExp(/^\(?(\d{2})\)?[- ]?(\d{3})[- ]?(\d{4})$/);
```

Comment:
So, here right the(70)-111-1111, but also right this:70-111-111, and right this 70111111 too. Every format is ok, just have nine decimal number. 
What will be the expected format?